Client: Implement NotifyPublish and NotifyReturn (#82)
0x4b53/amqp-rpc: README.md

    WithQueueDeclareSettings(QueueDeclareSettings{}).
    WithConsumeSettings(ConsumeSettings{}).
    WithPublishSettings(PublishSettings{}).
    WithConfirmMode(true),
    WithTimeout(10 * Time.Second)
```

> ### ConfirmMode

Reviewer: ```suggestion
### Confirm mode
```

> ConfirmMode can be set on the client and will make the client wait for `Ack`

Reviewer: ```suggestion
Confirm mode can be set on the client and will make the client wait for `Ack`
```

from the amqp-server. This makes sending requests more reliable at the cost of
some performance as each publishing must be confirmed by the amqp-server [Your
can read more here](https://www.rabbitmq.com/confirms.html#publisher-confirms)

> The Client is set in confirm-mode by default.

Reviewer: ```suggestion
The client is set in confirm mode by default.
```
